src/Gui/SoDatumLabel.cpp

```diff
@@ -59,6 +59,63 @@ using namespace Gui;
 
 // ------------------------------------------------------
 
+
+namespace glVec3f{
```

Comment:
Why this weird namespace name? Since you anyway import it below you can use an anonymous namespace.